### PR TITLE
Postgres setup

### PR DESCRIPTION
**Pull request by wontkim**
Installed postgres and set up connection along with seeding script for postgres.

### CODE REVIEW

File: public/helper-functions.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-const { Product } = require('../database/models/product.js', {
-  useNewUrlParser: true,
-  useUnifiedTopology: true,
-});
+// const { Product } = require('../database/models/product.js', {
+//   useNewUrlParser: true,
+//   useUnifiedTopology: true,
+// });
```
Comment (nathanq95) [inline public/helper-functions.js]: Don't forget to remove unneeded code

File: seeding/seedScriptPostgres.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+const brandName = [
+  'Lorem', 'ipsum', 'dolor', 'sit', 'amet', 'consectetur', 'adipiscing', 'elit'];
+
+const categoryName = ['Lorem', 'ipsum', 'dolor', 'sit', 'amet'];
+
+const playerCount = ['1 Player', '2 Player', '3 Player', '4 Player', '5 Player', '6 Player'];
+const partNumber = ['Lorem', 'ipsum', 'dolor', 'sit', 'amet', 'consectetur', 'adipiscing', 'elit', 'Sed', 'volutpat', 'feugiat', 'consequat', 'Donec', 'id', 'faucibus', 'ligula', 'Maecenas', 'lobortis', 'laoreet', 'maximus', 'Donec', 'vel', 'nunc', 'mi', 'Nam', 'rutrum', 'rutrum', 'convallis', 'Donec', 'tempus', 'gravida', 'risus', 'laoreet', 'elementum', 'Fusce', 'congue', 'rhoncus', 'sollicitudin', 'Nunc', 'quis'];
```
Comment (nathanq95) [inline seeding/seedScriptPostgres.js]: Consider storing these arrays in their own separate files so that could be reused for your secondary database.

**Review by nathanq95 — APPROVED** [review]
I just left a minor suggestion. Other than that everything's good!

File: seeding/seedScriptPostgres.js (hunk 2 above)
Comment (wontkim) [reply seeding/seedScriptPostgres.js]: Thanks. That's a good point. I refactored to use the original seeding functions.

**Pull request merged**